### ISSUE TITLE
Pre-Compiled Binaries would be super helpful

### ISSUE THREAD

**Issue by GrimblyGorn** [bmpix/PolyVectorization#6]
When I first found this I was super excited to have found it and play around with it. 

That was two days ago. 

I've spent the last two days very slowly downloading and installing all the required dependencies and building an Ubuntu VM around them just so I can compile a GUI .exe and CLI .exe 

But, that was pointless or perhaps didn't work out correctly cause now all I get is build errors at every turn. First, it was complaining about not having enough arguments for add_executable. Then it was giving me HermiteSpline errors as mentioned in Issue 3 so I did everything mentioned there to get past that. Now it's giving me a bunch of errors related to MainWindow from main.cpp I think.

This all seems very unreasonable to deal with if you already have a functional set up that can successfully compile these .exes

Not that it's required for you to provide those for us or anything. Rather, I'm simply saying if it's within the realm of possibility for you to do. It would be very helpful and highly appreciated if you did provide them for those of us that are not as well versed in making custom Linux builds and compiling things from the terminal.

**Comment by bmpix** [comment]
Sorry to hear you're having trouble! I'll commit a patch when I get to it. However, I don't think GUI.exe and CLI.exe are a part of this project. I am happy to provide binaries here, but I'm afraid those will be windows-only, I don't have a linux machine at hand. Will that work for you?

**Comment by GrimblyGorn** [comment]
Yes, that would be wonderful. I only went for a Linux VM build to avoid messing with my Windows build and because I had the ISO on hand already. I normally only use windows so having binaries for it would be perfect.

Thanks and thank you for the quick reply as well :)

**Comment by bmpix** [comment]
:) Deal! I will put the binaries on the main page, but in the meantime, here's an archive for you: https://www.dropbox.com/s/09bclqvm8xi1l11/polyvector_thing_binary.zip?dl=1

It should run right out of the box (sample inputs are included). If it doesn't, please lmk.

**Comment by GrimblyGorn** [comment]
Sorry for the delayed reply. I had to mow the lawn :)
 
I tried out the files you linked and they do work. Quite well in fact. I noticed it was even able to use both of my CPU's and seemingly all 32 cores from them so that was nice. I always like it when things can make use of all this hardware. I didn't check to see what, if anything, it was doing with the graphics card, but seeing as it completed my test runs on your sample images I'm sure it was able to use that fine too if it needed to.

Thanks again for making this available and being so helpful today. I'm playing around with it more now and testing out various samples of my own.
